CHANGELOG.md

```diff
@@ -2,10 +2,15 @@
 
 ## Unreleased
 
-* Set min SDK version to `2.9.0`
-* Changed the support version range of the analyzer to `>=0.40.0 <0.42.0`
-* Changed the support version range of the analyzer_plugin to `>=0.3.0 <0.5.0`
+* Set min `SDK` version to `2.9.0`
+* Set min `args` 1.6.0
+* Changed the support version range of the `analyzer` to `>=0.40.0 <0.42.0`
+* Changed the support version range of the `analyzer_plugin` to `>=0.3.0 <0.5.0`
 * Remove obsolete rule `prefer-trailing-comma-for-collection`
+* Rename cli arguments:
+  - `ignore-files` to `exclude`
```

Comment:
Hm... Maybe we should keep an alias for ignore-files for some time? Or some other way to notify users that this argument was renamed. 
Additionally, do we need some kind of a migration guide from 2.x.x to 3.0.0?